fv/bpf_test.go

```diff
@@ -266,6 +266,8 @@ func describeBPFTests(opts ...bpfTestOpt) bool {
 				options.WireguardEnabled = true
 				// Enable Wireguard.
 				options.ExtraEnvVars["FELIX_WIREGUARDENABLED"] = "true"
+				// With Wireguard the default IptablesMarkMask of 0xffff0000 isn't enough.
```

Comment:
Think we'll need to address that; maybe we ditch the high-order fixed bits in the BPF program marks?